### PR TITLE
Change BuildSnapshot Styling with onclick event

### PR DESCRIPTION
**Pull request by samuelsogbesan**
This PR adds a change that adjusts the orientation of the toggle icon and the composition of the borders for the BuildSnapshot Component. This is all bound together using a className that is conditioned to appear only when <code>props.isOpen === true</code>
- Smoothly rotate the toggle icon on click
- Reveal snapshot header border bottom on click.

### CODE REVIEW

File: src/main/webapp/style.css [hunk 1]
```diff
@@ -70,6 +70,15 @@ ul {
...
   width: 1.5rem;
+  transition: 300ms transform ease-in-out;
+}
+
+.snapshot-header.active > .header-toggle {
+  transform: rotate(90deg);
+}
+
+.snapshot-header.active {
+  border-bottom: 0.25rem var(--primary) solid;
```
Comment (MagoDelBlocco) [inline src/main/webapp/style.css:81]: nit: use .25 instead of 0.25

**Review by MagoDelBlocco — APPROVED** [review]
LGTM, left a nit

File: src/main/webapp/style.css (hunk 1 above)
Comment (samuelsogbesan) [reply src/main/webapp/style.css]: Adjust in all places! 46f2630

File: src/main/webapp/style.css [hunk 2]
```diff
@@ -70,6 +70,16 @@ ul {
 .header-toggle {
   height: 1.5rem;
   width: 1.5rem;
+  transition: 300ms transform ease-in-out;
+}
+
+/*Animate the open/close icon when snapshot is open*/
```
Comment (Somae) [inline src/main/webapp/style.css:76]:
```suggestion
/* Animate the open/close icon when snapshot is open */
```
See https://google.github.io/styleguide/htmlcssguide.html#Protocol for CSS comment examples with surrounding whitespaces.

File: src/main/webapp/style.css [hunk 3]
```diff
@@ -51,14 +51,14 @@ ul {
 
 .snapshot-header {
   align-items: center;
-  border: 0.25rem var(--primary) solid;
-  box-shadow: 0 0.25rem 0.25rem rgba(0, 0, 0, 0.25);
-  border-radius: 0.5rem;
+  border: .25rem var(--primary) solid;
+  box-shadow: 0 .25rem .25rem rgba(0, 0, 0, .25);
+  border-radius: .5rem;
```
Comment (Somae) [inline src/main/webapp/style.css:56]: Should this be in alphabetical order?

File: src/main/webapp/style.css (hunk 2 above)
Comment (Somae) [reply src/main/webapp/style.css:76]: Please also add what kind of animation it is. Animate the [] by [] for example.

**Review by Somae — CHANGES_REQUESTED** [review]

File: src/main/webapp/style.css (hunk 2 above)
Comment (samuelsogbesan) [reply src/main/webapp/style.css]: I've updated the comment to be more specific d8a307d

File: src/main/webapp/style.css (hunk 3 above)
Comment (samuelsogbesan) [reply src/main/webapp/style.css:55]: Silly mistake! d8a307d

**Review by Somae — APPROVED** [review]

**Pull request merged**